diff --git a/content/07710/assets/mermaid.md b/content/07710/assets/diagram.mmd
similarity index 97%
rename from content/07710/assets/mermaid.md
rename to content/07710/assets/diagram.mmd
index d2a0ad5e..ef9f79a1 100644
--- a/content/07710/assets/mermaid.md
+++ b/content/07710/assets/diagram.mmd
@@ -1,4 +1,3 @@
-```mermaid
 flowchart LR
     redeemer(["redeemer"]) --"redeemDelegations"--> Delegation_Manager(["Delegation Manager"])
     Delegation_Manager -.->|validate delegation w/ Action| Delegation_Manager
@@ -7,4 +6,3 @@ flowchart LR
     classDef action stroke:#333,stroke-width:2px,stroke-dasharray: 5, 5;
     classDef entity fill:#af,stroke:#333,stroke-width:2px;
     class redeemer,Delegator,Delegation_Manager,Target entity;
-```
